entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
```

Comment:
We can probably remove the `-x` part? (Though thankfully it doesn’t print the `$SSH_PRIVATE_KEY` anyways, apparently.)

Reply:
Can do; I'd put it there to make debugging easy and left it there for the same reason. Happy to remove though